### PR TITLE
More Obvious OSM Search Input Fields

### PR DESCRIPTION
**Pull request by dakeyras7**
This PR fixes https://trello.com/c/XmHtYynd/17-improve-event-editor-form

Unfortunately the file has not been prettier-formatted before and thus the diff looks quite heavy.
But the changes were quite minimal.

* The prop `placeholder={t`Type to search on OpenStreetMap…`}` was updated to tell the user more directly to type into the field
* The `dropdownIndicator` and `indicatorSeparator` parts were style-overriden to be invisible when no value was entered yet into the input field. There is small helper function `styleHideOverride` for this.

### CODE REVIEW

File: imports/client/components/Forms/LocationPicker/OSMPickerField.tsx [hunk 1]
```diff
@@ -52,51 +47,75 @@ const UnstyledOSMPicker = (
...
+  return (
+    <div className={className}>
+      <label htmlFor={props.id} className={props.error ? 'text-danger' : ''}>
+        {props.label}
+      </label>
+      <AsyncSelect
+        className={`form-control react-select ${props.error ? 'is-invalid' : ''}`}
+        noOptionsMessage={s => (s ? t`No locations found` : t`Type to search`)}
+        loadingMessage={s => t`Loading…`}
+        placeholder={t`Type to search on OpenStreetMap…`}
```
Comment (mutaphysis) [inline imports/client/components/Forms/LocationPicker/OSMPickerField.tsx]: -1: Please update translation files & translate german string on transifex.

**Comment by dakeyras7** [comment]
I used `meteor npm install`. Let me try `meteor npm ci` and see if something changed. Didn't know that, sorry.

**Comment by mutaphysis** [comment]
> I used `meteor npm install`. Let me try `meteor npm ci` and see if something changed. Didn't know that, sorry.

No, don't worry - minor version changes should be fine, most of the time 👍

File: imports/client/components/Forms/LocationPicker/OSMPickerField.tsx (hunk 1 above)
Comment (dakeyras7) [reply imports/client/components/Forms/LocationPicker/OSMPickerField.tsx]: With translation files you mean the commit `*.po` files correct?

Comment (dakeyras7) [reply imports/client/components/Forms/LocationPicker/OSMPickerField.tsx]: Done

**Comment by dakeyras7** [comment]
I did the whole transifex process, translated the new string and fetched the latest `de` and `en` translations and commited those as well.

**Pull request merged**